src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -0,0 +1,75 @@
+package com.zendesk.maxwell;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class MaxwellConfig {
+	public String mysqlHost;
+	public int    mysqlPort;
+	public String mysqlUser;
+	public String mysqlPassword;
+
+	public String currentPositionFile;
+
+	private BinlogPosition initialPosition;
+
+	public Connection getMasterConnection() throws SQLException {
+		return DriverManager.getConnection("jdbc:mysql://" + mysqlHost + ":" + mysqlPort, mysqlUser, mysqlPassword);
+	}
+
+	public BinlogPosition getInitialPosition() throws FileNotFoundException, IOException {
+		if ( this.initialPosition != null )
+			return this.initialPosition;
+
+		File f = new File(this.currentPositionFile);
+		if ( !f.exists() ) {
+			return null;
+		} else {
+			Properties p = new Properties();
+			p.load(new FileReader(f));
+
+			this.initialPosition = new BinlogPosition(Integer.valueOf((String) p.get("offset")), p.getProperty("file"));
+			return this.initialPosition;
+		}
+	}
+
+	public void setInitialPosition(BinlogPosition position) throws IOException {
+		Properties p = new Properties();
+		p.setProperty("offset", String.valueOf(position.getOffset()));
+		p.setProperty("file", position.getFile());
+
+		File f = new File(this.currentPositionFile);
+		FileWriter fw = new FileWriter(f);
+		try {
+			p.store(fw, "");
+			this.initialPosition = position;
+		} finally {
+			fw.close();
+		}
+	}
```

Comment:
haven't 100% decided where to store last-position.  probably mysql, but we might also just write the file/position into the kafka stream and pop the last-written off the stream to recover the position...